components/automate-cli/pkg/verifyserver/constants/externalpostgresqlconstants.go

```diff
@@ -0,0 +1,9 @@
+package constants
+
+const (
+	EXTERNALPGSUCCESSCONNECTIONTITLE  = "Machine is able to connect with External Managed Postgres"
+	EXTERNALPGFAILCONNECTIONTITLE     = "External Postgresql Connection failed"
```

Comment:
EXTERNALPGFAILCONNECTIONTITLE = "External Postgresql Connection failed"
it should be 
EXTERNAL_PG_FAIL_CONNECTION_TITLE = "External PostgreSQL connection failed"